packages/react-core/package.json

```diff
@@ -25,6 +25,7 @@
   },
   "homepage": "https://github.com/patternfly/patternfly-react#readme",
   "dependencies": {
+    "@patternfly/patternfly-next": "^1.0.11",
```

Comment:
This needs to stay in devDependencies, It is not required for users to consume this project. We also want it to be locked.  It will make keeping track of changes easier for now. Since core is still publishing breaking changes, still alpha, this will save us some headaches.